lib/src/command/add.dart

```diff
@@ -40,16 +39,16 @@ class AddCommand extends PubCommand {
   @override
   String get docUrl => 'https://dart.dev/tools/pub/cmd/pub-add';
   @override
-  bool get isOffline => argResults['offline'];
+  bool? get isOffline => argResults['offline'];
```

Comment:
All these args should cast to non-null if there is a default value.
```suggestion
  bool get isOffline => argResults['offline'] as bool;
```